tests/test_phabricator_client.py

```diff
@@ -16,6 +16,8 @@
 from tests.canned_responses.phabricator.repos import *
 from tests.canned_responses.phabricator.errors import *
 
+pytestmark = pytest.mark.usefixtures('docker_env_vars')
```

Comment:
Ignore the specifics above, but, otherwise is there a better way to have this be done automatically for every test?